docs/reference/saving_hooks.rst

```diff
@@ -31,7 +31,7 @@ Example used with the FOS/UserBundle
 ------------------------------------
 
 The ``FOSUserBundle`` provides authentication features for your Symfony Project,
-and is compatible with Doctrine ORM, Doctrine ODM and Propel. See
+and is compatible with Doctrine ORM, Doctrine ODM. See
```

Comment:
this talks about FOSUserBundle, but they removed propel too from the main package: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2321